executor/distsql.go

```diff
@@ -43,6 +44,8 @@ const (
 	minLogDuration = 50 * time.Millisecond
 )
 
+var xIndexSelectGP = gp.New(3 * time.Minute)
```

Comment:
why use diffrent idleTimeout?

Reply:
It doesn't matter whether it 2mins or 3mins or anything.